editor/animation_track_editor.cpp

```diff
@@ -6083,6 +6215,16 @@ AnimationTrackEditor::AnimationTrackEditor() {
 	track_vbox->add_child(track_copy_select);
 	track_copy_dialog->connect("confirmed", this, "_edit_menu_pressed", varray(EDIT_COPY_TRACKS_CONFIRM));
 	animation_changing_awaiting_update = false;
+
+	track_duplicate_dialog = memnew(ConfirmationDialog);
+	add_child(track_duplicate_dialog);
+	track_duplicate_dialog->set_title(TTR("Select tracks to duplicate:"));
+	track_duplicate_dialog->get_ok()->set_text(TTR("Duplicate"));
+
+	track_duplicate_select = memnew(Tree);
+	track_duplicate_select->set_hide_root(true);
+	track_duplicate_dialog->add_child(track_duplicate_select);
+	track_duplicate_dialog->connect("confirmed", this, "_edit_menu_pressed", varray(EDIT_DUPLICATE_TRACKS_CONFIRM));
```

Comment:
While rebasing, this will need to be changed to use `callable_mp`:
```cpp
track_duplicate_dialog->connect("confirmed", callable_mp(this, &AnimationTrackEditor::_edit_menu_pressed), varray(EDIT_DUPLICATE_TRACKS_CONFIRM));
```